client/blocks/inline-help/inline-help-forum-guide.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import { preventWidows } from 'lib/formatting';
+
+const FORUM_LINK = '//en.forums.wordpress.com';
+
+const InlineHelpRichForumGuide = ( { translate = noop } ) => (
```

Comment:
Would we want this the be `InlineHelpForumGuide?`

Reply:
Good spot - a very silly typo! 😆